### PR TITLE
🧹 Removed old root UI components.

### PR DESCRIPTION
**Pull request by JeroenReumkens**
## What does this PR do?

Removes old root UI components. 

**Environment**: Staging(main branch)

## Type of change
- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

There shouldn't be any functionality touched, so as long as the build succeeds, everything should be fine 🔥 Mainly deleted old code and replaced a few icons with the new import.

### CODE REVIEW

**Comment by PeerRich** [comment]
should we refactor all `<Icon.FiName>` to `<Icon.Name>`?

**Comment by JeroenReumkens** [comment]
> should we refactor all `<Icon.FiName>` to `<Icon.Name>`?

@PeerRich  Definitely a cool idea! But I can not come up with a smarter way of doing that, than manually creating an object that maps FiName to Name and exporting that. It DOES however give us more control over which icons we want people to use. So perhaps it isn't even a bad idea to do that way.

File: packages/ui/components/buttonGroup/buttongroup.stories.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
-import { Examples, Example, Note, Title,CustomArgsTable,VariantsTable,VariantsRow } from '@calcom/storybook/components'
```
Comment (JeroenReumkens) [inline packages/ui/components/buttonGroup/buttongroup.stories.mdx:2]: This was a typo, file didn't exist.

File: packages/ui/components/icon/Icon.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { CollectionIcon } from "@heroicons/react/outline";
+import { ShieldCheckIcon } from "@heroicons/react/outline";
+import { BadgeCheckIcon } from "@heroicons/react/outline";
+import { ClipboardCopyIcon } from "@heroicons/react/outline";
+import { StarIcon as StarIconSolid } from "@heroicons/react/solid";
+// find all feather icons at https://feathericons.com/
+// github https://github.com/feathericons/feather
+import * as ReactIcons from "react-icons/fi";
+
+export const Icon = {
```
Comment (JeroenReumkens) [inline packages/ui/components/icon/Icon.tsx:11]: Merge reacticons and custom feather icons together in one Icon import.

File: packages/ui/form/fields.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import classNames from "@calcom/lib/classNames";
 import { getErrorFromUnknown } from "@calcom/lib/errors";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 
-import { Alert } from "../Alert";
+import { Alert } from "../v2/core/Alert";
```
Comment (JeroenReumkens) [inline packages/ui/form/fields.tsx:9]: Path will be fixed once alert is migrated.

File: packages/ui/components/icon/Icon.tsx (hunk 2 above)
Comment (sean-brydon) [reply packages/ui/components/icon/Icon.tsx:11]: Smart

**Review by sean-brydon — APPROVED** [review]
LGTM - tested most pages on dev and it seems to be working fine! No more icon error either

**Pull request merged**